tests/e2e/unified_test.go

```diff
@@ -375,6 +375,25 @@ func testFixturesInSeries(ctx context.Context, t *testing.T, testPause bool, can
 						return false
 					})
 
+					// Remove repeated GET requests
+					{
+						var previous *test.LogEntry
+						events = events.KeepIf(func(e *test.LogEntry) bool {
+							keep := true
+							if e.Request.Method == "GET" && previous != nil {
+								if previous.Request.Method == "GET" && previous.Request.URL == e.Request.URL {
+									if previous.Response.Status == e.Response.Status {
+										if previous.Response.Body == e.Response.Body {
```

Comment:
Is that possible the response body is the same but reordered? I encountered this before, so that I [sort JSON ](https://github.com/GoogleCloudPlatform/k8s-config-connector/blob/master/tests/e2e/unified_test.go#L645) before comparing.

Reply:
I guess it's possible, but I don't think I've encountered it yet (_most_ JSON libraries sort the keys I believe).  Let's tackle it when we do?